canonical/ubuntu.com
# Footer refresh

## Done

- refreshed footer as per [design](https://www.figma.com/file/Le4TTzyOFbcbqTPJ1tVoZT/U.com---Home-(rebranding)?type=design&node-id=2910-7036&mode=design&t=WNhKAX8o0kJZuiXp-4) and [doc](https://docs.google.com/document/d/117orMiMtXPBwowp6sjJ1ygjROYjSLnUkvosac7nqmpw/edit)

## QA

- View the site locally in your web browser at: https://ubuntu-com-13825.demos.haus/
- Check against design and doc
  - Be sure to check smaller screens as well 

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-10375
